<launch>

    <arg name="num_agents" default="1"/>

    <node pkg="vswarm" name="gazebo_mocap_node" type="gazebo_mocap_node" output="screen">
        <param name="num_agents" value="$(arg num_agents)" />
    </node>

</launch>